src/audio-worklet/free-queue/example/README.md

```diff
@@ -0,0 +1,44 @@
+# Free Queue C
+
+Example of Using FreeQueue with WebAssembly App written in C.
+
+
+
+## Building
+
+Install and Activate Emscripten SDK and use
+
+`build.sh` for Linux 
+
+or
+
+`build.cmd` for Windows
+
+or
+
+Makefile
+
+or
+
+```ps
+emcc src/main.c -sENVIRONMENT=worker -sMODULARIZE=1 -sEXPORT_NAME=FQC -sEXPORT_ES6=1 -sINVOKE_RUN=0 -pthread -sEXPORTED_RUNTIME_METHODS="['callMain','ccall', 'cwrap']" -o build/main.js
```

Comment:
This shouldn't be necessary because it's already in the makefile. Also the instruction should have "CMake" as a prerequisite for the local development.